cel2sql_test.go

```diff
@@ -31,6 +31,7 @@ func TestConvert(t *testing.T) {
 			decls.NewVar("height", decls.Double),
 			decls.NewVar("string_list", decls.NewListType(decls.String)),
 			decls.NewVar("string_int_map", decls.NewMapType(decls.String, decls.Int)),
+			decls.NewVar("nullable_string", decls.NewWrapperType(decls.String)),
```

Comment:
Please add a test for nullable bytes as well.

Reply:
done